exercises/rotational-cipher/example.py

```diff
@@ -0,0 +1,12 @@
+def rotational_cipher(message, key):
+    import string
+    alpha_lower = list(string.ascii_lowercase)
+    alpha_upper = list(string.ascii_uppercase)
+    coded_message = ""
+    for char in list(message):
+        if char in alpha_lower:
+            char = alpha_lower[(alpha_lower.index(char) + key) % 26]
```

Comment:
You could create a constant for the length of alphabet `ALPHA_LEN = len(alpha_lower)`. In this case it would also work if the ascii alphabet is changed, which is totally unlikely to happen. So, using 26 is fine as well.